hera_cal/tests/test_io.py

```diff
@@ -331,6 +331,8 @@ def test_build_datacontainers(self):
             assert isinstance(dc, DataContainer)
             for k in dc.antpos.keys():
                 assert np.all(dc.antpos[k] == hd.antpos[k])
+            for k in dc.data_antpos.keys():
+                assert np.all(dc.data_antpos[k] == hd.data_antpos[k])
```

Comment:
I know this may be trivial, but technically don't we also want a test that does a down-select and shows that len(data_ants) < len(ants)?

Reply:
The test doesn't actually do a downselect, it's just making sure the attribute is there and it matches between the HERAData object and the datacontainer

Reply:
Good point. Done